modules/govuk/manifests/deploy/config.pp

```diff
@@ -129,14 +134,15 @@
     'RACK_ENV':  value => $govuk_env;
     'RAILS_ENV': value => $govuk_env;
 
-    'ERRBIT_ENVIRONMENT_NAME': value => $errbit_environment_name;
-    'SENTRY_CURRENT_ENV': value      => $errbit_environment_name;
-    'GOVUK_ENVIRONMENT_NAME': value  => $govuk_environment_name;
-    'GOVUK_ASSET_ROOT': value        => $asset_root;
-    'GOVUK_WEBSITE_ROOT': value      => $website_root;
-    'GOVUK_CSP_REPORT_ONLY': value   => $csp_report_only_value;
-    'GOVUK_CSP_REPORT_URI': value    => $csp_report_uri;
-    'SIDEKIQ_LOGFILE': value         => $sidekiq_logfile;
+    'ERRBIT_ENVIRONMENT_NAME': value    => $errbit_environment_name;
+    'SENTRY_CURRENT_ENV': value         => $errbit_environment_name;
+    'GOVUK_ENVIRONMENT_NAME': value     => $govuk_environment_name;
+    'GOVUK_ASSET_ROOT': value           => $asset_root;
+    'GOVUK_WEBSITE_ROOT': value         => $website_root;
+    'GOVUK_CSP_REPORT_ONLY': value      => $csp_report_only_value;
+    'GOVUK_CSP_REPORT_URI': value       => $csp_report_uri;
+    'SIDEKIQ_LOGFILE': value            => $sidekiq_logfile;
+    'GOVUK_SIDEKIQ_JSON_LOGGING': value => $govuk_sidekiq_json_logging
```

Comment:
```suggestion
    'GOVUK_SIDEKIQ_JSON_LOGGING': value => '1'
```

We've not actually set up a `$govuk_sidekiq_json_logging` variable. I don't think we really need to have it as a variable though as we just want it set everywhere.